rancher.py

```diff
@@ -11,24 +11,54 @@
 import requests
 from OpenSSL import crypto
 from requests.auth import HTTPBasicAuth
+import sys
 
 try:
+    # These variables should all get set as this service is a Rancher Agent
+    # Therefore, Rancher sets these for us.
     RANCHER_URL = os.environ['CATTLE_URL']
     RANCHER_ACCESS_KEY = os.environ['CATTLE_ACCESS_KEY']
     RANCHER_SECRET_KEY = os.environ['CATTLE_SECRET_KEY']
+
+    # list of domains we want certs for, comma-delimited
     DOMAINS = os.environ['DOMAINS']
     # convert renew days -> seconds
```

Comment:
If we removed the constants doing this conversion, can probably lose the line comment as well. Or at least move it to part of the block statement below ...